src/lib/slide-toggle/slide-toggle.ts

```diff
@@ -83,13 +83,13 @@ export class MdSlideToggle implements AfterContentInit, ControlValueAccessor {
   @Input() id: string = this._uniqueId;
 
   /** Used to specify the tabIndex value for the underlying input element. */
-  @Input() tabIndex: number = 0;
+  @Input() tabindex: number = 0;
```

Comment:
@jelbourn Changed it back to `tabIndex`, because I think that the consistency with the `HTMLElement` property is more weighted than the `tabindex` attribute.

> Didn't change it for the deprecated input component, because making some breaking changes on something that's deprecated feels wrong.